src/index.tsx

```diff
@@ -826,6 +826,98 @@ export function useBandwidthEstimation() {
   return { estimation };
 }
 
+/**
+ * This hook provides access to current rtc statistics data.
+ */
+export function useRTCStatistics() {
+  const MAX_SIZE = 120;
+  const [statistics, setStatistics] = useState<object[]>([]);
+
+  useEffect(() => {
+    const eventListener = eventEmitter.addListener(
+      'StatisticsUpdated',
+      (stats) => {
+        if (statistics.length === MAX_SIZE) {
```

Comment:
check this out: https://lodash.com/docs/4.17.15#takeRight 
also you can do something like this:
`[...statistics, processIncomingStats(stats)]`